lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        *positions*
-            a sequence of numerical values or a 1D numpy array.  Can be None
+        Parameters
+        ----------
+        positions : 1D array-like object or None.
```

Comment:
Done (reverted my changes and instead simply fixed the relevant docstring) by fc212ee.